testing/config.go

```diff
@@ -22,13 +22,12 @@ type TendermintConfig struct {
 	IsWasmClient    bool
```

Comment:
We can remove this too?

Were all of the other fields also removed? I believe there was something attached to the `Coorindator` struct specfic to wasm clients also